<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OverlayUI</class>
 <widget class="QDialog" name="OverlayUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Overlay</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../ldforge.qrc">
    <normaloff>:/icons/overlay.png</normaloff>:/icons/overlay.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Camera</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="top">
        <property name="text">
         <string>Top</string>
        </property>
        <property name="icon">
         <iconset resource="../../ldforge.qrc">
          <normaloff>:/icons/camera-top.png</normaloff>:/icons/camera-top.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="front">
        <property name="text">
         <string>Front</string>
        </property>
        <property name="icon">
         <iconset resource="../../ldforge.qrc">
          <normaloff>:/icons/camera-front.png</normaloff>:/icons/camera-front.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="left">
        <property name="text">
         <string>Left</string>
        </property>
        <property name="icon">
         <iconset resource="../../ldforge.qrc">
          <normaloff>:/icons/camera-left.png</normaloff>:/icons/camera-left.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="bottom">
        <property name="text">
         <string>Bottom</string>
        </property>
        <property name="icon">
         <iconset resource="../../ldforge.qrc">
          <normaloff>:/icons/camera-bottom.png</normaloff>:/icons/camera-bottom.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="back">
        <property name="text">
         <string>Back</string>
        </property>
        <property name="icon">
         <iconset resource="../../ldforge.qrc">
          <normaloff>:/icons/camera-back.png</normaloff>:/icons/camera-back.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QRadioButton" name="right">
        <property name="text">
         <string>Right</string>
        </property>
        <property name="icon">
         <iconset resource="../../ldforge.qrc">
          <normaloff>:/icons/camera-right.png</normaloff>:/icons/camera-right.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Image</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::ExpandingFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>File:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="filename"/>
          </item>
          <item>
           <widget class="QPushButton" name="fileSearchButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../ldforge.qrc">
              <normaloff>:/icons/file-open.png</normaloff>:/icons/file-open.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Origin:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QSpinBox" name="originX">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="suffix">
             <string> px</string>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="originY">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="suffix">
             <string> px</string>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Dimensions:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QDoubleSpinBox" name="width">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="suffix">
             <string> LDU</string>
            </property>
            <property name="maximum">
             <double>10000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="height">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="suffix">
             <string> LDU</string>
            </property>
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10000.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../ldforge.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OverlayUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OverlayUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
